similarity/config/configurations.properties

```diff
@@ -0,0 +1,20 @@
+wordMetric	paragram
```

Comment:
It does extend Configurator class. See the SimConfigurator class here https://github.com/lingss0918/cogcomp-nlp/blob/master/similarity/src/main/java/edu/illinois/cs/cogcomp/config/SimConfigurator.java